src/dealer-list.js

```diff
@@ -44,18 +44,27 @@ const DealerCard = (dealer, key, onDealerClicked, theme) => (
     </Dealer>
 );
 
+const filterByOnlineDealer = dealer => {
+  if (dealer.online) {
+    return dealer;
+  }
+};
+
 const DealerList = ({
   dealers,
   onDealerClicked,
   mapCenter,
   mapBoundary,
   border,
   theme,
-  dealerCardComponent
+  dealerCardComponent,
+  filterOnlineDealers
 }) => {
-  const dealersOnMap = _.filter(dealers, (dealer) =>
-    isLocationWithinBoundary(dealer.location, mapBoundary)
-  );
+  const DealerFilterProp = filterOnlineDealers;
```

Comment:
I'd consider renaming this variable to something like "OnlineOnly" or something. There are multiple layers of filtering that could be happening, but this one seems like it only applies to that online flag.